### PR TITLE
Add API localized error messages

### PR DESCRIPTION
**Pull request by maxxcrawford**

### CODE REVIEW

File: en/app.ftl [hunk 1]
```diff
@@ -1143,3 +1143,17 @@ forwarded-email-footer-offer-countdown-banner = Upgrade to { $premium_link } bef
...
+# Variables:
+#   $free_tier_limit (string) - Maxmimum email masks created for free account
+# api-error-free-tier-limit = You must be a premium subscriber to make more than { $free_tier_limit } email masks.
+api-error-free-tier-limit = You must be a premium subscriber to make more than 5 email masks.
+api-error-free-tier-no-subdomain-masks = You must be a premium subscriber to create subdomain email masks.
+
+# Variables:
+#   $unavailable_address (string) - User-set subdomain that is not allowed
+api-error-address-unavailable = Domain address "{ $unavailable_address }" could not be created, try using a different value.
+api-error-need-subdomain = You must select a subdomain before creating email address with subdomain.
```
Comment (jwhitlock) [inline en/app.ftl:1158]:
```suggestion
api-error-need-subdomain = You must select a subdomain before creating a domain address.
```

File: en/app.ftl [hunk 2]
```diff
@@ -1143,3 +1143,17 @@ forwarded-email-footer-offer-countdown-banner = Upgrade to { $premium_link } bef
 ## Notifications component
 
 toast-button-close-label = Close notification
+
+## API error messages
+
+# Variables:
+#   $free_tier_limit (string) - Maxmimum email masks created for free account
```
Comment (peiying2) [inline en/app.ftl:1150]: Is this a string or a number?

File: en/app.ftl [hunk 3]
```diff
@@ -1143,3 +1143,17 @@ forwarded-email-footer-offer-countdown-banner = Upgrade to { $premium_link } bef
...
+# api-error-free-tier-limit = You must be a premium subscriber to make more than { $free_tier_limit } email masks.
```
Comment (peiying2) [inline en/app.ftl:1151]: is this a comment or a string?

**Review by peiying2 — CHANGES_REQUESTED** [review]

File: en/app.ftl (hunk 1 above)
Comment (maxxcrawford) [reply en/app.ftl]: New string added from Mel: 
> Please select a subdomain before creating a custom email address.

File: en/app.ftl (hunk 2 above)
Comment (maxxcrawford) [reply en/app.ftl]: I had this PR as a draft. I should've added some sort of "Not ready for review" label. 🙈

File: en/app.ftl (hunk 3 above)
Comment (maxxcrawford) [reply en/app.ftl]: It was a comment. This PR was still in draft mode. Disregarding comment for now.

**Review by maxxcrawford — COMMENTED** [review]
This is ready for review now. Thanks!

File: en/app.ftl [hunk 4]
```diff
@@ -1143,3 +1143,16 @@ forwarded-email-footer-offer-countdown-banner = Upgrade to { $premium_link } bef
 ## Notifications component
 
 toast-button-close-label = Close notification
+
+## API error messages
+
+# Variables:
+#   $free_tier_limit (number) - Maxmimum email masks created for free account
+api-error-free-tier-limit = You’ve used all { $free_tier_limit } email masks included with your free account. You can reuse an existing mask, but using a unique mask for each account is the most secure option.
+api-error-free-tier-no-subdomain-masks = Your free account does not include custom subdomains for masks. To create custom masks, upgrade to Relay Premium.
```
Comment (Vinnl) [inline en/app.ftl:1152]: Sorry, happened to see this PR :)

```suggestion
api-error-free-tier-no-subdomain-masks = Your free account does not include custom subdomains for masks. To create custom masks, upgrade to { -brand-name-relay-premium }.
```

**Review by peiying2 — APPROVED** [review]

**Review by jwhitlock — CHANGES_REQUESTED** [review]
Thanks @maxxcrawford, it is great to get these updated to current terminology and translated!

I agree w/ @Vinnl about using the `-brand-name-relay-premium` placeholder. I'm not familiar with the Fluent syntax, so I'm 90% he got it right.

**Comment by Vinnl** [comment]
@maxxcrawford Whenever you're ready to merge this, can you merge https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/114 at the same time to minimise notifications for our localisers? Thanks!

File: en/app.ftl (hunk 4 above)
Comment (maxxcrawford) [reply en/app.ftl]: Great catch, @Vinnl!

**Comment by maxxcrawford** [comment]
Merged into other open PR to reduce `main` branch merge noise.

**Pull request merged**